### ISSUE TITLE
Rsync error after doing vagrant up

### ISSUE THREAD

**Issue by maxpaynestory** [Sylius/Vagrant#4]
"rsync" could not be found on your PATH. Make sure that rsync
is properly installed on your system and available on the PATH.

On Windows 10, virtual Box

**Comment by pix-art** [comment]
This is an issue with your vagrant setup. You'll have to ask the vagrant guys for more help. This setup is made with nfs for linux/mac users. 

Hope this helps you enough.
Joeri

**Comment by maxpaynestory** [comment]
Yes, i figured it out long ago, i used different vagrant box and sylius installed on it successfully. Thank you for the clarification. You may close this issue.

**Comment by pix-art** [comment]
Sorry for the late response :-)

**Comment by maxpaynestory** [comment]
I wish if could initiate a pull request but unfortunately i don't have a mac / linux host machines to test my vagrant box.

**Comment by pix-art** [comment]
Feel free to do a pull request and we could ask others to test it.

**Comment by mablae** [comment]
Just had another strange rsync related issue and wanted to share it here for others:
https://github.com/hashicorp/vagrant/issues/6702#issuecomment-240367451:
> this is broken with the SSH client that comes in Git Bash.

Fixed it by installing Cygwin64 `rsync` + `ssh` add add that `bin/` dir to `$PATH`

**Comment by Bonobomagno** [comment]
yep, still broken on windows.
